.github/workflows/build.yml

```diff
@@ -13,6 +13,7 @@ env:
   ECR_REPO: '720541322708.dkr.ecr.eu-west-2.amazonaws.com/medic'
   ECR_PUBLIC_REPO: 'public.ecr.aws/s5s3h4s7'
   COUCHDB_LOG_LEVEL: 'debug'
+  TAG: ${{ (github.ref_type == 'tag' && github.ref_name) || '' }}
```

Comment:
If I'm reading this right, all the code in the `scripts` dir is correct, we just needed to pass in the TAG?

Reply:
Yup, I think that at some point the `TAG` was populated by something (there were already references to this) but then it got removed in some update to GH builds or something. 

I tested this on a fork of mine: 
https://github.com/dianabarsan/cht-core/actions/runs/3640208802/jobs/6144573769#step:7:20 

I just added a build job that prints the result of `getVersion` and pushed a tag. The output was the tag name itself.